test/testsupport/testdata/TtpJsonResponses.scala

```diff
@@ -87,6 +88,20 @@ object TtpJsonResponses {
        |""".stripMargin
   }
 
+  def optionalEligibilityResponsesJson(eligibilityRules: EligibilityRules): String = {
```

Comment:
you can perhaps save some repetition by having a method which looks something ilke this:
```scala
def optionalEligibilityResponsesJson(flag: Option[Boolean], flagName: String): String =
  flag.fold(""){f => s""",  "$flagName": ${f.toString} """   }
 ```
and then call it like:
```scala
optionalEligibilityResponsesJson(eligibilityRules.hasInvalidInterestSignals, "hasInvalidInterestSignals")
```
etc.

Reply:
Sure thing, will update